src/ClassicUO.Client/LegionScripting/docs/PyBaseControl.md

```
@@ -28,6 +27,18 @@ description: PyBaseControl class documentation
 
 ---
 
+### Add
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)